### DISCUSSION TITLE
FF7R cheats wrong version

### DISCUSSION THREAD

**Discussion by subworx** [GoldHEN/GoldHEN_Cheat_Repository#15] [Q&A] [answered]
Hi, I tested around with the Final Fantasy VII Remake cheats.

CUSA07211 (US) and CUSA07187 (EU) v1.00, 1.01 and 1.02.
The database only has one entry for 07211, v1.0**1**, I renamed the json files on PS4 to match all 6 variants but didn't change anything else.
Only tested the first three cheats in the json (Inf. HP, Inf. MP, Inf ATB).

07211 1.00: works
07211 1.01: not working
07211 1.02: not working
07187 1.00: works
07187 1.01: not working
07187 1.02: not working

So the cheats can (if someone verifies, please) be relabeled from 1.01 compatible to 1.00 compatible, but added to CUSA07187 as well.

Thanks and regards.

**Comment by kmeps4** [comment]
Hello. Thx for the report.. will take a look

**Comment by kmeps4** [comment, marked as answer]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094
